runatlantis.io/guide/test-drive.md

```diff
@@ -7,12 +7,15 @@ Once you've extracted the archive, run:
 ./atlantis testdrive
 ```
 
-This mode sets up Atlantis on a test repo so you can try it out. It will
+This mode sets up Atlantis on a test repo so you can try it out. It will:
 - Fork an example Terraform project into your GitHub account
 - Install Terraform (if not already in your PATH)
-- Install [ngrok](https://ngrok.com/) so we can expose Atlantis to GitHub
+- Install [Tunnelmole](https://tunnelmole.com/docs) or [ngrok](https://ngrok.com/) to expose Atlantis to GitHub. Tunnelmole is a free and open source tunneling tool while Ngrok is a popular closed source tunneling tool.
 - Start Atlantis so you can execute commands on the pull request
 
+## Tunnelmole Setup
+First install Tunnelmole. You can check out the [Installation Guide](https://tunnelmole.com/docs/#installation). For most use cases, this will be done with a single command copy/pasted into your terminal. Running Tunnelmole can be as simple as `tmole <port>`(replace `<port>` with your listening port). In the output, you'll find an HTTP and HTTPS URL. It's recommended to use the HTTPS URL for improved privacy and security.
+
```

Comment:
I would suggest removing this paragraph, as there is no ngrok setup detail in this document, so having this extra Tunnelmole detail here is confusing here if you intend to use ngrok. You already have links to the tunnelmole docs and the additional setup detail in the `testling-locally.md` file.

Reply:
Good point, I've removed it.